### PR TITLE
Various DIM updates

### PR DESCRIPTION
**Pull request by timbitss**
- Individual board RGB LED set to correct colour
- Set error table periodically within can rx thread
- Turn on red IMD and BSPD LEDs if IMD OK and BMS OK CAN signals from the BMS respectively are false
- AMS/BMS RGB LED only turns red if BMS_OK CAN signal is false, otherwise it is off

### Testing Done
None yet

### CODE REVIEW

File: boards/DIM/Src/App/states/App_DriveState.c [hunk 1]
```diff
@@ -151,6 +161,11 @@ static void DriveStateRunOnTick100Hz(struct StateMachine *const state_machine)
 
     for (size_t i = 0; i < NUM_BOARDS; i++)
     {
+        if (i == BMS)
+        {
+            continue;
+        }
```
Comment (pasitlaothamatas) [inline boards/DIM/Src/App/states/App_DriveState.c:167]: which spec is this implementing?

**Review by Eddy13013 — APPROVED** [review]
Approved, but we should verify this on the car first.

File: boards/DIM/Src/App/states/App_DriveState.c (hunk 1 above)
Comment (timbitss) [reply boards/DIM/Src/App/states/App_DriveState.c:167]: AMS OK indicator light

**Comment by timbitss** [comment]
tested

**Pull request merged**